Updates for Top SQL GA (#7914)
pingcap/docs: system-variables.md

- Default value: `OFF`
- This variable is used to control whether to enable the [Top SQL](/dashboard/top-sql.md) feature.

> ### `tidb_top_sql_max_meta_count` (New in v6.0.0)

Author: ```suggestion
### tidb_top_sql_max_meta_count (New in v6.0.0)
```

- Scope: GLOBAL
- Default value: `5000`
- Range: `[1, 10000]`
- This variable is used to control the maximum number of SQL statement types collected by [Top SQL](/dashboard/top-sql.md) per minute.

> ### `tidb_top_sql_max_time_series_count` (New in v6.0.0)

Author: ```suggestion
### tidb_top_sql_max_time_series_count (New in v6.0.0)
```

- Scope: GLOBAL
- Default value: `100`
- Range: `[1, 5000]`

> - This variable is used to control [Top SQL](/dashboard/top-sql.md) can record how many SQL statements that contribute the most to the load (that is, top N) per minute.

Author: ```suggestion
- This variable is used to control how many SQL statements that contribute the most to the load (that is, top N) can be recorded by [Top SQL](/dashboard/top-sql.md) per minute.
```
